plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/PinotMetadata.java

```diff
@@ -151,7 +151,7 @@ public List<String> listSchemaNames(ConnectorSession session)
     @Override
     public PinotTableHandle getTableHandle(ConnectorSession session, SchemaTableName tableName)
     {
-        if (tableName.getTableName().trim().startsWith("select ")) {
+        if (tableName.getTableName().trim().contains("select ")) {
```

Comment:
This would allow table names like "fooselect ", etc. which would be a syntax error. Why was this changed?

Reply:
Yeah, my feeling is that as long as it's a sql statement, then it's still going to the same code block for execution. We can add a query parser to fail fast, but not a must.